src/app/components/layout/certificate-base-login/certificate-base-login.component.ts

```diff
@@ -69,14 +69,18 @@ export class CertificateBaseLoginComponent implements OnInit {
 
   LoginWithCertificate() {
     if (this.certificateContent.length <= 0
-      && this.form.value.certificateText.length <= 0) {
+      && this.form.get('certificateText').value.length <= 0) {
       this.alertService.error('Certificate is required');
       return;
     }
     if (this.certificateContent.length <= 0
-      && this.form.value.certificateText !== '') {
-      this.certificateContent = this.form.value.certificateText;
+      && this.form.get('certificateText').value !== '') {
+      this.certificateContent = this.form.get('certificateText').value;
     }
+    if (this.form.get('certificateText').value !== '' && this.certificateContent.length > 0) {
+      this.certificateContent = this.form.get('certificateText').value;
+    }
```

Comment:
Can you please put a expected behavior comment here?. **Looks** if manually put content option is chosen and there is some value in textarea then browsed file shall be ignored even if that was done before toggling the option.